### ISSUE TITLE
Difference between cordova-plugin-mobile-ocr and cordova-plugin-ml-text

### ISSUE THREAD

**Issue by Catscratch** [NeutrinosPlatform/cordova-plugin-mobile-ocr#44]
Since this plugin seems outdated, is [cordova-plugin-ml-text](https://github.com/NeutrinosPlatform/cordova-plugin-ml-text) the current solution to use? What's the difference?

Thanks!

**Comment by ChrisTomAlx** [comment]
Tl;Dr :
Use the [cordova-plugin-ml-text](https://github.com/NeutrinosPlatform/cordova-plugin-ml-text). It is more recent, but less mature since it has not been used as much as the ocr plugin.

Long version :
The ml-text plugin uses the more recent ml-kit based libraries. The mobile-ocr plugin was created when those libraries were in its infancy. It wasn't really a kit back then. I created the ml-kit plugin when google moved the libraries expecting this plugin to be outdated at some point. Although these plugins are similar their dependencies are slightly varied.

Cheers,
[Chris](https://github.com/ChrisTomAlx)
[Neutrinos](http://www.neutrinos.co/)
